renovate-configs/dee-platform-ops.json5

```diff
@@ -0,0 +1,86 @@
+// Shared Renovate Base Configuration for repos operated by the
+// @GoogleCloudPlatform/dee-platform-ops team.
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+  ],
+  // Rate limits, to protect maintainers (and CI) from PR floods.
+  "prHourlyLimit": 10,
+  "prConcurrentLimit": 30,
+  "minimumReleaseAge": "7 days",
+  "vulnerabilityAlerts":{
+     "labels":[
+       "type:security"
+     ],
+     "minimumReleaseAge": null
+  },
+  "labels": ["dependencies"],
+  // Dependency Dashboard should be excluded from SLO calculation.
+  "dependencyDashboardLabels": [
+    "type: process",
+  ],
+  "packageRules": [
+    // Label PRs based on the languages they contain
+    // category ids are listed at https://docs.renovatebot.com/modules/manager/
+    {"matchCategories": ["python"], "addLabels": ["lang: python"]},
+    {"matchCategories": ["java"], "addLabels": ["lang: java"]},
+    {"matchCategories": ["golang"], "addLabels": ["lang: go"]},
+    {"matchCategories": ["js"], "addLabels": ["lang: nodejs"]},
+    {"matchCategories": ["dotnet"], "addLabels": ["lang: dotnet"]},
```

Comment:
**Not action needed:** Just so you're aware. We also have a:
* one-off Ruby sample inside the GKE samples repo: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/quickstart/ruby/Gemfile.
* one-off PHP sample inside the GKE samples repo: https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/guestbook/php-redis/composer.json

Sounds like Renovate [supports PHP](https://docs.renovatebot.com/php/) and [supports Ruby](https://docs.renovatebot.com/ruby/).
But I don't think we need to be that thorough. Up to you.

Reply:
I was planning to do this in a future iteration - I'll add it as future work to the proposal so we don't lose track of it.

Another benefit to using a shared config is we can update this in one place, and see updates in all our repos 😄